docs/quality-assurance/safety-assurance-guidance.md

```diff
@@ -0,0 +1,25 @@
+# Safety Assurance Guidelines
+
+## What is Safety Modelling
+
+Safety Modelling is focused around two key areas:
+
+* Error (Data Integrity)
+* Delay
+
+This is centered around asking *What if* questions to determine safety risks and the likelihood and safety impact of those risks occurring.
+
+## How should we use it
+
+### At the start of a project
+
+* Carry out a bowtie analysis of the project to identify potential areas of safety that need to be considered (there are a number of safety assurance experts who can help in carrying out bowtie analysis).
```

Comment:
Is it worth adding a brief description of what "bowtie" is?

Reply:
Good idea - added.